cmd/fleet/serve_test.go

```diff
@@ -91,6 +91,12 @@ func TestMaybeSendStatistics(t *testing.T) {
 			LicenseTier:                          "premium",
 			NumHostsEnrolled:                     999,
 			NumUsers:                             99,
+			NumSoftwares:                         100,
```

Comment:
```suggestion
			NumSoftwareVersions:                     101,
```

Small nit. Suggest being consistent w/ "software versions" naming we use in the API: https://fleetdm.com/docs/rest-api/rest-api#list-software-versions

Also what's the difference between `NumSoftware` and `NumHostSoftware`?

Reply:
- NumSoftware: from the `software` table, holds all the software in the Fleet regardless of where it's installed (what we list in Software tab).
- NumHostSoftware: from `host_software` table, holds which hosts the software in `software` is installed in (what we list in the Host Details Software tab).

It's important to know these numbers for close-to-reality load tests:
Usually `host_software` is a few orders of magnitude larger than `software` (e.g. if you have VSCode version X installed on 100 hosts, then you'll have one row in `software` but 100 rows in `host_software`), but we'd like to have more precise numbers from real deployments. E.g. in dogfood we have ~9k software rows, and ~30k host software rows.